pages/mobile/account_settings.py

```diff
@@ -46,7 +46,7 @@ def click_sign_in(self):
     def click_my_apps(self, logged_in=True):
         self.scroll_to_element(self._settings_options_locator[0], self._settings_options_locator[1] % ("/purchases"))
         self.selenium.find_element(self._settings_options_locator[0], self._settings_options_locator[1] % ("/purchases")).click()
-        if logged_in == True:
+        if logged_in is True:
```

Comment:
Is the strict equality check needed? Could this just be `if logged_in:` instead?